mappings/net/minecraft/client/network/ClientPlayerEntity.mapping

```diff
@@ -42,8 +54,11 @@ CLASS net/minecraft/class_746 net/minecraft/client/network/ClientPlayerEntity
 	METHOD method_3136 sendMovementPackets ()V
 	METHOD method_3137 closeScreen ()V
 	METHOD method_3138 updateHealth (F)V
+		ARG 1 newHealth
+	METHOD method_3140 getVisibilityReductionFromWater ()F
```

Comment:
Hmm, is this like some opacity? if yes we can use shorter terms

Reply:
See discussion on `wateriness`